payroll-processor-functions/src/payroll-processor-functions/Features/Employees/EmployeeEntity.cs

```diff
@@ -0,0 +1,63 @@
+using System;
+using System.Collections.Generic;
+using Microsoft.WindowsAzure.Storage.Table;
+using Newtonsoft.Json;
+
+namespace PayrollProcessor.Functions.Features.Employees
+{
+    /// <summary>
+    /// The table storage representation of an employee
+    /// </summary>
+    public class EmployeeEntity : TableEntity
+    {
+        public string Department { get; set; } = "";
+        public DateTimeOffset EmploymentStartedOn { get; set; }
+        public string FirstName { get; set; } = "";
+        public string LastName { get; set; } = "";
+        public string Phone { get; set; } = "";
+        public string Status { get; set; } = "";
+        public string Title { get; set; } = "";
+        public string Payrolls { get; set; } = "";
+
+        public static class Map
+        {
+            public static Employee To(EmployeeEntity entity)
+            {
+                var payrolls = JsonConvert.DeserializeObject<IEnumerable<EmployeePayroll>>(entity.Payrolls);
+
+                return new Employee(Guid.Parse(entity.RowKey))
+                {
+                    Department = entity.Department,
+                    EmploymentStartedOn = entity.EmploymentStartedOn,
+                    FirstName = entity.FirstName,
+                    LastName = entity.LastName,
+                    Phone = entity.Phone,
+                    Status = entity.Status,
+                    Title = entity.Status,
```

Comment:
Status maps to Title and Status in this method

Reply:
Good catch! Fixed